src/validator.cpp

```diff
@@ -944,7 +946,7 @@ void Validator::ValidatorImpl::validateAndCleanMathCiCnNodes(XmlNodePtr &node, c
                 // Check for a matching standard units.
                 if (!isStandardUnitName(unitsName)) {
                     ErrorPtr err = std::make_shared<Error>();
-                    err->setDescription("Math has a " + nodeType + " element with a cellml:units attribute '" + unitsName +
+                    err->setDescription("Math has a " + node->getType() + " element with a cellml:units attribute '" + unitsName +
```

Comment:
Does getType() output include a namespace? Doesn't matter right now, but could cause confusion if we ever have overlapping element names